tests/acceptance/features/coreApiWebdavUploadTUS/checksums.feature

```diff
@@ -1,11 +1,14 @@
 @api
 Feature: checksums
+  As a user
+  I want to upload resources with checksum
+  So that I can detect errors while uploading resources
```

Comment:
Writing feature description for some API tests is tricky. I couldn't think of any good description.
we should look at what the scenarios are doing in the When and Then steps.

Reply:
can we write something like `So that the integrity of the uploaded resources is maintained`?

Reply:
This "so that" seems reasonable.